tests/integrations/raftstore/test_flashback.rs

```diff
@@ -0,0 +1,203 @@
+// Copyright 2022 TiKV Project Authors. Licensed under Apache-2.0.
+
+use std::time::Duration;
+
+use engine_traits::CF_DEFAULT;
+use test_raftstore::*;
+use txn_types::WriteBatchFlags;
+
+#[test]
+fn test_flahsback_for_parameters() {
+    let mut cluster = new_node_cluster(0, 3);
+    cluster.pd_client.disable_default_operator();
+
+    cluster.run();
+    cluster.must_put(b"k1", b"v1");
+
+    let region = cluster.get_region(b"k1");
+    cluster.call_prepare_flashback(region.get_id(), 1);
+    sleep_ms(100);
+
+    assert!(
+        cluster.store_metas[&1]
+            .lock()
+            .unwrap()
+            .readers
+            .get(&region.get_id())
+            .unwrap()
+            .flashback_pending_ignore
+    );
+
+    cluster.call_finish_flashback(region.get_id(), 1);
+    sleep_ms(100);
+
+    assert!(
+        !cluster.store_metas[&1]
+            .lock()
+            .unwrap()
+            .readers
+            .get(&region.get_id())
+            .unwrap()
+            .flashback_pending_ignore
+    );
+}
+
+#[test]
+fn test_flahsback_for_write() {
+    let mut cluster = new_node_cluster(0, 3);
+    cluster.pd_client.disable_default_operator();
+    cluster.run();
+
+    let reqs = vec![new_put_cf_cmd(CF_DEFAULT, b"k1", b"v1")];
+    let resp = cluster.request(b"k1", reqs, false, Duration::from_secs(5));
+    assert!(!resp.get_header().has_error());
+
+    let mut region = cluster.get_region(b"k1");
+    cluster.call_prepare_flashback(region.get_id(), 1);
+    sleep_ms(1000);
```

Comment:
We should wait on the channel returned by `call_prepare_flashback` rather than sleeping here.